SU2_CFD/src/solver_adjoint_mean_inc.cpp

```diff
@@ -3490,51 +3489,11 @@ void CAdjIncEulerSolver::BC_Outlet(CGeometry *geometry, CSolver **solver_contain
       case MASS_FLOW_RATE:
         Psi_outlet[0]+=1;
         break;
-      case OUTFLOW_GENERALIZED:
```

Comment:
I will remove this shortly. I think these were copied over when someone that *_inc file was created - these boundary conditions would NOT be expected to work with the incompressible solver; there is literature on incompressible adjoints for a few objectives, and the equations will be different.